components/studio/libexec/hab-studio-type-default.sh

```diff
@@ -84,6 +86,12 @@ exec $HAB_ROOT_PATH/bin/hab pkg exec core/hab-plan-build hab-plan-build \$*
 EOF
   $bb chmod $v 755 $HAB_STUDIO_ROOT$HAB_ROOT_PATH/bin/build
 
+  $bb cat <<TAIL_SUP > $HAB_STUDIO_ROOT$HAB_ROOT_PATH/bin/tail_sup
```

Comment:
This can't become a command to `hab-sup` because we don't actually log to file by convention. We're just redirecting output here in the studio to provide this particular functionality - so it would make sense that this stay a command of the studio